cpp/util/bignum_test.cc

```diff
@@ -0,0 +1,194 @@
+#include "util/bignum.h"
+
+#include <gtest/gtest.h>
+
+#include "util/testing.h"
+
+
+namespace cert_trans {
+
+
+class BigNumTest : public ::testing::Test {};
+
+
+TEST_F(BigNumTest, TestDefaultCtor) {
+  BigNum n;
+  EXPECT_EQ(n, 0);
+}
+
+
+TEST_F(BigNumTest, TestWordCtor) {
+  const int kValue(0x1234);
```

Comment:
Just 'cos I didn't really care about the size and I knew I was going to be boring and pick small test values (resting my weary fingers etc. etc.), can change them if you like?